pkg/download/list_merge_test.go

```diff
@@ -0,0 +1,142 @@
+package download
+
+import (
+	"bytes"
+	"context"
+	"errors"
+	"io/ioutil"
+	"testing"
+
+	athenser "github.com/gomods/athens/pkg/errors"
```

Comment:
I like this name - maybe members of the athens community should be called "athenians" but also we could call ourselves "athensers"

😆